cluster-service/deploy/openshift-templates/arohcp-service-template.yml

```
@@ -256,6 +262,26 @@ objects:
       usePodIdentity: "false"
     provider: azure
 
+- apiVersion: secrets-store.csi.x-k8s.io/v1
+  kind: SecretProviderClass
+  metadata:
+    name: arm-identity
+    namespace: ${NAMESPACE}
+  spec:
+    parameters:
+      clientID: ${ARM_HELPER_IDENTITY_CLIENT_ID}
+      cloudName: AzurePublicCloud
+      keyvaultName: ${ARM_HELPER_IDENTITY_KEYVAULT_NAME}
```

```suggestion
      keyvaultName: ${SERVICE_KEYVAULT_NAME}
```
The `ARM_HELPER_IDENTITY_KEYVAULT_NAME` does not exist after your changes.